README.md

```diff
@@ -79,6 +79,11 @@ The generated extensions allow composition of proto messages in a dsl style. Sup
 ```
 ---
 
+Note: while naming your proto files, only snake_case is supported:
+* `some_service.proto` - snake_case, good
+* `some-service.proto` - kebab-case, bad
+* `someService.proto` - camelCase, bad
+
```

Comment:
While it is correct that protobuf best practices is to use snake case for file naming conventions, the protoc java and grpc plugin support malformed file names and so should we. If there a particular example of a file name that produced an invalid output?  

I know in the past we had this issue with with some malformed services names. The plugin was updated and test were added to verify those cases.

Reply:
Yes, using both `kebab-case` or `camelCase` produces incorrect class names in `RpcOverloads`.
If you expected it to work, I can add issue/test/fix, but for now it doesn't work.